backend/src/routes/doctorOrders.js

```diff
@@ -225,18 +225,20 @@ router.delete('/api/deleteAll', async (req, res) => {
   res.send([]);
 });
 
-const getRemsAdminFhirUrl = order => {
+const getRemsAdminEtasuUrl = order => {
   const rxnorm = order.drugRxnormCode;
   const remsDrug = medicationRequestToRemsAdmins.find(entry => {
     return Number(rxnorm) === Number(entry.rxnorm);
   });
-  return remsDrug?.remsAdminFhirUrl;
+  const baseUrl = remsDrug?.remsAdminFhirUrl;
+  const remsAdminFhirUrl = baseUrl + '/GuidanceResponse/$rems-etasu';
+  return baseUrl ? remsAdminFhirUrl : null;
 };
 
 const getGuidanceResponse = async order => {
-  const remsAdminFhirUrl = getRemsAdminFhirUrl(order);
+  const etasuUrl = env.USE_INTERMEDIARY ? env.INTERMEDIARY_ETASU_MET : getRemsAdminEtasuUrl(order);
```

Comment:
I think you should move this logic into the getRemsAdminUrl function above and add on the /GuidanceResponse/$rems-etasu in that function to every FHIR URL including what is set for the intermediary.